homeassistant/components/sabnzbd/sensor.py

```diff
@@ -23,13 +23,14 @@ class SabnzbdSensor(SensorEntity):
 
     def __init__(self, sensor_type, sabnzbd_api_data, client_name):
         """Initialize the sensor."""
-        self._client_name = client_name
-        self._field_name = SENSOR_TYPES[sensor_type][2]
-        self._name = SENSOR_TYPES[sensor_type][0]
+        meta_data = SENSOR_TYPES[sensor_type]
+        self._field_name = meta_data.key
         self._sabnzbd_api = sabnzbd_api_data
-        self._state = None
+        self._attr_state = None
         self._type = sensor_type
```

Comment:
I think we can loose this type and use the `key` of the entity description instead

Reply:
Done